Upstream-Status: Pending

automake 1.12 has decpricated automatic de-ANSI-fication support

this patch fixes this build issue:

| configure.in:67: error: automatic de-ANSI-fication support has been removed
Signed-Off-by: Nitin A Kamble <nitin.a.kamble@intel.com>
2012/05/02

Index: libxml2-2.7.8/configure.in
===================================================================
--- libxml2-2.7.8.orig/configure.in
+++ libxml2-2.7.8/configure.in
@@ -63,8 +63,6 @@ AC_PATH_PROG(WGET, wget, /usr/bin/wget)
 AC_PATH_PROG(XMLLINT, xmllint, /usr/bin/xmllint)
 AC_PATH_PROG(XSLTPROC, xsltproc, /usr/bin/xsltproc)
 
-dnl Make sure we have an ANSI compiler
-AM_C_PROTOTYPES
 test "x$U" != "x" && AC_MSG_ERROR(Compiler not ANSI compliant)
 
 AC_LIBTOOL_WIN32_DLL
